client/main.lua

```diff
@@ -605,7 +606,8 @@ RegisterNetEvent('qb-doorlock:client:addNewDoor', function()
 			type = "setText",
 			aim = "block"
 		})
-		local entity, coords, heading, model, result, entityHit = 0, 0, 0, 0, false, 0
+		local heading, result, entityHit
```

Comment:
That entityHit doesn't have a value could possible create errors, so please make sure that has the value of `0`, otherwise it'll try to compare nil to a number

Reply:
Instead of assigning it a 0 on start (which causes linting error: value assigned but not used), I've added a second return parameter which ensures that it always has a value, either 0 or a valid one